<ui version="4.0" >
 <class>Tano</class>
 <widget class="QMainWindow" name="Tano" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Tano Player</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="images.qrc" >
    <normaloff>:/icons/images/tano.png</normaloff>:/icons/images/tano.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" name="verticalLayout" >
    <item>
     <widget class="QGroupBox" name="videoBox" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title" >
       <string>Video</string>
      </property>
      <layout class="QVBoxLayout" name="_2" >
       <item>
        <widget class="Video" native="1" name="videoWidget" />
       </item>
       <item>
        <widget class="QFrame" name="frame" >
         <property name="minimumSize" >
          <size>
           <width>0</width>
           <height>64</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>16777215</width>
           <height>64</height>
          </size>
         </property>
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="videoControls" >
          <property name="topMargin" >
           <number>2</number>
          </property>
          <property name="bottomMargin" >
           <number>2</number>
          </property>
          <item>
           <widget class="QPushButton" name="buttonPlay" >
            <property name="minimumSize" >
             <size>
              <width>46</width>
              <height>46</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>46</width>
              <height>46</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Play</string>
            </property>
            <property name="icon" >
             <iconset resource="images.qrc" >
              <normaloff>:/icons/images/player_play.png</normaloff>:/icons/images/player_play.png</iconset>
            </property>
            <property name="iconSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonStop" >
            <property name="minimumSize" >
             <size>
              <width>46</width>
              <height>46</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>46</width>
              <height>46</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Stop</string>
            </property>
            <property name="icon" >
             <iconset resource="images.qrc" >
              <normaloff>:/icons/images/player_stop.png</normaloff>:/icons/images/player_stop.png</iconset>
            </property>
            <property name="iconSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonFull" >
            <property name="minimumSize" >
             <size>
              <width>46</width>
              <height>46</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>46</width>
              <height>46</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Fullscreen</string>
            </property>
            <property name="icon" >
             <iconset resource="images.qrc" >
              <normaloff>:/icons/images/player_fullscreen.png</normaloff>:/icons/images/player_fullscreen.png</iconset>
            </property>
            <property name="iconSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="shortcut" >
             <string>Ctrl+F</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonVlc" >
            <property name="minimumSize" >
             <size>
              <width>46</width>
              <height>46</height>
             </size>
            </property>
            <property name="maximumSize" >
             <size>
              <width>46</width>
              <height>46</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Play with VLC</string>
            </property>
            <property name="icon" >
             <iconset resource="images.qrc" >
              <normaloff>:/icons/images/vlc.png</normaloff>:/icons/images/vlc.png</iconset>
            </property>
            <property name="iconSize" >
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0" >
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenFile" />
    <addaction name="actionOpen" />
    <addaction name="separator" />
    <addaction name="actionBrowser" />
    <addaction name="separator" />
    <addaction name="actionClose" />
   </widget>
   <widget class="QMenu" name="menuAbout" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelp" />
    <addaction name="separator" />
    <addaction name="actionAbout" />
    <addaction name="actionAbout_Qt" />
   </widget>
   <widget class="QMenu" name="menuOptions" >
    <property name="title" >
     <string>&amp;Tools</string>
    </property>
    <widget class="QMenu" name="menuShow" >
     <property name="title" >
      <string>Show</string>
     </property>
     <property name="icon" >
      <iconset resource="images.qrc" >
       <normaloff>:/icons/images/win.png</normaloff>:/icons/images/win.png</iconset>
     </property>
     <addaction name="actionChannel_list" />
     <addaction name="actionChannel_info_EPG" />
    </widget>
    <addaction name="actionFullscreen" />
    <addaction name="menuShow" />
    <addaction name="separator" />
    <addaction name="actionEditPlaylist" />
    <addaction name="actionSettings" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuOptions" />
   <addaction name="menuAbout" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="iconSize" >
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle" >
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenFile" />
   <addaction name="actionOpen" />
   <addaction name="separator" />
   <addaction name="actionBrowser" />
   <addaction name="separator" />
   <addaction name="actionEditPlaylist" />
   <addaction name="actionSettings" />
  </widget>
  <widget class="QDockWidget" name="playlistWidget" >
   <property name="floating" >
    <bool>false</bool>
   </property>
   <property name="features" >
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle" >
    <string>Channel list</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QVBoxLayout" name="verticalLayout_4" >
     <item>
      <widget class="QTreeWidget" name="playlistTree" >
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="animated" >
        <bool>true</bool>
       </property>
       <property name="columnCount" >
        <number>1</number>
       </property>
       <column>
        <property name="text" >
         <string>Channel</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="epgWidget" >
   <property name="floating" >
    <bool>false</bool>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2" >
    <layout class="QVBoxLayout" name="verticalLayout_3" >
     <property name="spacing" >
      <number>4</number>
     </property>
     <property name="margin" >
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="labelLanguage" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutNow" >
       <item>
        <widget class="QLabel" name="labelNow" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonRefresh" >
         <property name="maximumSize" >
          <size>
           <width>200</width>
           <height>24</height>
          </size>
         </property>
         <property name="text" >
          <string>Refresh Currently playing</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonRefreshToday" >
         <property name="maximumSize" >
          <size>
           <width>200</width>
           <height>24</height>
          </size>
         </property>
         <property name="text" >
          <string>Refresh Today</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="verticalScrollBarPolicy" >
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable" >
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>902</width>
          <height>50</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" >
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="labelTodayEpg" >
           <property name="minimumSize" >
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAbout" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/tano.png</normaloff>:/icons/images/tano.png</iconset>
   </property>
   <property name="text" >
    <string>About Tano Player</string>
   </property>
  </action>
  <action name="actionAbout_Qt" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/qt.png</normaloff>:/icons/images/qt.png</iconset>
   </property>
   <property name="text" >
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionClose" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/exit.png</normaloff>:/icons/images/exit.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionSettings" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/settings.png</normaloff>:/icons/images/settings.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionFullscreen" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/player_fullscreen.png</normaloff>:/icons/images/player_fullscreen.png</iconset>
   </property>
   <property name="text" >
    <string>Fullscreen</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+F</string>
   </property>
   <property name="shortcutContext" >
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/fileopen.png</normaloff>:/icons/images/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Open Channel list</string>
   </property>
  </action>
  <action name="actionOpenFile" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/fileopen.png</normaloff>:/icons/images/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
  <action name="actionBrowser" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/browser.png</normaloff>:/icons/images/browser.png</iconset>
   </property>
   <property name="text" >
    <string>EPG Browser</string>
   </property>
  </action>
  <action name="actionEditPlaylist" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/edit.png</normaloff>:/icons/images/edit.png</iconset>
   </property>
   <property name="text" >
    <string>Edit Channel list</string>
   </property>
  </action>
  <action name="actionHelp" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/help.png</normaloff>:/icons/images/help.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Help</string>
   </property>
   <property name="shortcut" >
    <string>F1</string>
   </property>
   <property name="shortcutContext" >
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionChannel_list" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/folder_video.png</normaloff>:/icons/images/folder_video.png</iconset>
   </property>
   <property name="text" >
    <string>Channel list</string>
   </property>
  </action>
  <action name="actionChannel_info_EPG" >
   <property name="icon" >
    <iconset resource="images.qrc" >
     <normaloff>:/icons/images/browser.png</normaloff>:/icons/images/browser.png</iconset>
   </property>
   <property name="text" >
    <string>Channel info / EPG</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Video</class>
   <extends>QWidget</extends>
   <header location="global" >video/Video.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>actionChannel_list</sender>
   <signal>triggered()</signal>
   <receiver>playlistWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>785</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChannel_info_EPG</sender>
   <signal>triggered()</signal>
   <receiver>epgWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel" >
     <x>458</x>
     <y>643</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
